### ISSUE TITLE
[Bug] Extension needs to be set to run on local

### ISSUE THREAD

**Issue by TanakritBenz** [sedders123/vscode-ytmusic#55]
Hi, thank you for building the YT Music extension. I started using it a week ago and it's been working great!

Due to my VSCode setup being a remote dev setup, one minor issue I had was that I needed to explicitly tell VSCode to run this extension locally.
```
  "remote.extensionKind": {
    "sedders123.vscode-ytmusic": ["ui"]
  },
```
And I got this temporary fix from: https://code.visualstudio.com/docs/remote/ssh#_advanced-forcing-an-extension-to-run-locally-remotely

--------------------------------------------------------------

I've never developed any VSCode extension before but skimming through the doc, I think all you need to do is:

> If the Developer: Show Running Extensions command shows that a UI extension is incorrectly being treated as a workspace extension or vice versa, try setting the extensionKind property in your extension's [package.json](https://code.visualstudio.com/api/get-started/extension-anatomy#extension-manifest) as described in the [Extension Kinds section](https://code.visualstudio.com/api/advanced-topics/extension-host#preferred-extension-location).

Source: https://code.visualstudio.com/api/advanced-topics/remote-extensions#incorrect-execution-location
This should then make your extension run on local by default.

**Comment by sedders123** [comment]
Hi @TanakritBenz, thanks so much for reporting this issue! I've just published Version 1.2.2 which should include the fix for this. Please do re open this issue if after upgrading it's still not behaving correctly :)

**Comment by TanakritBenz** [comment]
Yup, that did the trick! Thank you for such a quick turnaround 😄
